core/DataAccess/LogAggregator.php

```diff
@@ -421,7 +422,7 @@ public function generateQuery($select, $from, $where, $groupBy, $orderBy, $limit
         return $query;
     }
 
-    protected function getVisitsMetricFields()
+    public function getVisitsMetricFields()
```

Comment:
Any reason for making this public? Couldn't find any additionally usage that requires it 🤔

Reply:
I was using it, forgot to change it back :+1: